torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -204,6 +220,17 @@ export async function getTestOwnerLabels(testFile: string): Promise<string[]> {
   }
 }
 
+
+export function wasRecent(test: FlakyTestData) {
+  if (test.eventTimes) {
+    return test.eventTimes.some(
+      (value) => dayjs().diff(dayjs(value), "minutes") < 3 * 60
```

Comment:
Where is this magic number 3 come from?  Is this the same as the `NUM_HOURS` constant?

Reply:
oops